kedro/framework/cli/micropkg.py

```diff
@@ -354,7 +354,7 @@ def _refactor_code_for_unpacking(
 
     to:
     <temp_dir>  # also the root of the Rope project
-    |__ <project_package>
+    |__ <project-name>
```

Comment:
```suggestion
    |__ <package_name>
```

Reply:
Changes made in [344a6fb](https://github.com/kedro-org/kedro/pull/1753/commits/344a6fb49d232371b8f592ee5ccc35d681bee043).